creusot-contracts/src/std/ptr.rs

```diff
@@ -116,27 +167,78 @@ impl<T: ?Sized> PointerExt<T> for *mut T {
     fn is_null_logic(self) -> bool {
         self.addr_logic() == 0usize
     }
+
+    #[logic(open)]
```

Comment:
```suggestion
    #[logic(open, inline)]
```

(This will help you through `compute_specify`, which behaves a bit like `simpl` in Rocq.)